bridge-abi/Forc.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 authors = ["Fuel Labs <contact@fuel.sh>"]
-entry = "bridge_fungible_token_abi.sw"
+entry = "bridge_abi.sw"
```

Comment:
I kinda feel like `bridge_abi` is too generic because there will eventually be a non-fungible bride as well that will need a different abi